shuup/admin/forms/_base.py

```diff
@@ -20,3 +21,12 @@ def __init__(self, **kwargs):
             if issubclass(self.fields[field].widget.__class__, AdminFileWidget):
                 self.fields[field].widget = FileDnDUploaderWidget(
                     upload_path="/default", kind="images", clearable=True)
+
+
+class ShuupAdminFormNoTranslation(ModelForm):
```

Comment:
DRY. This code is the same as the class above, you should create some mixin for this